cluster/gce/gci/configure.sh

```diff
@@ -236,20 +236,24 @@ function install-node-problem-detector {
 
 function install-cni-binaries {
   if [[ -n "${CNI_VERSION:-}" ]]; then
-      local -r cni_tar="cni-plugins-amd64-${CNI_VERSION}.tgz"
+      local -r cni_tar="cni-plugins-linux-amd64-${CNI_VERSION}.tgz"
+      local -r cni_version="${CNI_VERSION}"
       local -r cni_sha1="${CNI_SHA1}"
   else
-      local -r cni_tar="cni-plugins-amd64-${DEFAULT_CNI_VERSION}.tgz"
+      local -r cni_tar="cni-plugins-linux-amd64-${DEFAULT_CNI_VERSION}.tgz"
+      local -r cni_version="${DEFAULT_CNI_VERSION}"
       local -r cni_sha1="${DEFAULT_CNI_SHA1}"
   fi
   if is-preloaded "${cni_tar}" "${cni_sha1}"; then
     echo "${cni_tar} is preloaded."
     return
   fi
 
-  echo "Downloading cni binaries"
-  download-or-bust "${cni_sha1}" "${CNI_STORAGE_PATH}/${cni_tar}"
+  local -r cni_download_url="${CNI_GITHUB_DOWNLOAD_BASE_URL}/${cni_version}/${cni_tar}"
```

Comment:
> CNI_GITHUB_DOWNLOAD_BASE_URL="${CNI_GITHUB_DOWNLOAD_BASE_URL:-https://github.com/containernetworking/plugins/releases/download/}"

should we remove the `/` before ${cni_version} to not have double slashes in the URL?

Reply:
Thanks for catching that, @aojea. Just fixed it!